src/views/projects/ProjectCreate.vue

```diff
@@ -90,9 +109,7 @@ export default {
     ...mapGetters(['currentOrg', 'currentProject']),
 
     canProgress() {
-      return [this.projectName, this.dateFormat, this.projectFormat].every(
-        (field) => field && field.length > 0,
-      );
+      return true;
```

Comment:
If always true, you can remove it as a computed property

Reply:
Done ✅! Thanks!